sites/public/src/pages/account/application/[id]/lottery-results.tsx

```diff
@@ -39,6 +41,14 @@ export default () => {
                 .publicLotteryResults({ id: applicationId })
                 .then((results) => {
                   setResults(results)
+                  lotteryService
```

Comment:
This network call chain has become pretty intense. Since these calls don't actually rely on the previous data in order to run do we have to chain it like this? Some of these calls could be done simultaneously. For example we have the listing id after fetching the application and don't need to wait for the lottery results to do that call.

Reply:
Also, we are retrieving more data than we need to. Unless I'm missing something it looks like we are just getting the application in order to get the listing id. I'm wondering if there is some consolidation we can do with the 4 endpoints. Or in other words would it be a viable solution to create one endpoint just for this page? I don't remember where the conversations ended on that discussion

Reply:
Agreed on the nesting and the endpoint consolidation, that just felt like it could turn into a bigger refactor to undertake here. Having one endpoint here I think also sounds ideal! If that feels blocking for this PR happy to do it now.

Reply:
This is not blocking, but we should probably create a ticket to address it at some point